packages/app-project/src/screens/ClassifyPage/components/YourStats/components/DailyClassificationsChart/DailyClassificationsChartContainer.js

```diff
@@ -1,56 +1,44 @@
 import counterpart from 'counterpart'
-import { inject, observer } from 'mobx-react'
 import { array, number, shape, string } from 'prop-types'
-import { Component } from 'react'
 
 import DailyClassificationsChart from './DailyClassificationsChart'
 
-function storeMapper (stores) {
-  const { project, user: { personalization: { counts, stats: { thisWeek } } } } = stores.store
-  return {
-    counts,
-    thisWeek,
-    projectName: project['display_name']
-  }
+const defaultCounts = {
+  today: 0
 }
 
-@inject(storeMapper)
-@observer
-class DailyClassificationsChartContainer extends Component {
-  render () {
-    const TODAY = new Date()
-    const { counts, projectName, thisWeek } = this.props
-    const stats = thisWeek.map(stat => {
-      const day = new Date(stat.period)
-      const locale = counterpart.getLocale()
-      const count = (day.getDay() === TODAY.getDay()) ? counts.today : stat.count
-      const longLabel = day.toLocaleDateString(locale, { weekday: 'long' })
-      const alt = `${longLabel}: ${count}`
-      const label = day.toLocaleDateString(locale, { weekday: 'narrow' })
-      return Object.assign({}, stat, { alt, count, label, longLabel })
-    })
-    return (
-      <DailyClassificationsChart
-        stats={stats}
-        projectName={projectName}
-      />
-    )
-  }
+function DailyClassificationsChartContainer({
+  counts = defaultCounts,
+  projectName,
+  thisWeek = []
+}) {
+  const TODAY = new Date()
+  const stats = thisWeek.map(stat => {
+    const day = new Date(stat.period)
+    const locale = counterpart.getLocale()
+    const count = (day.getUTCDay() === TODAY.getUTCDay()) ? counts.today : stat.count
+    const longLabel = day.toLocaleDateString(locale, { weekday: 'long' })
```

Comment:
This might need to add the `timeZone` option, to override the local clock.

Reply:
Yep. I changed my Mac's time zone (in date and time preferences) and the graph labels changed to the local clock, one day off from the expected day names. We should be using UTC for the datetimes sent back from the stats API, local clock otherwise.